easybuild/easyconfigs/d/DISCOVAR/DISCOVARdenovo-52488-foss-2015a.eb

```diff
@@ -0,0 +1,22 @@
+name = 'DISCOVARdenovo'
+version = '52488'
+
+homepage = 'http://www.broadinstitute.org/software/discovar/blog/'
+description = """DISCOVAR de novo can generate de novo assemblies for both large and small genomes. It currently does not call variants."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = ['%(namelower)s-%(version)s.tar.gz']
+source_urls = ['ftp://ftp.broadinstitute.org/pub/crd/DiscovarDeNovo/latest_source_code']
+
+dependencies = [
+    ('GMP', '6.0.0a'),
```

Comment:
This was not the reason, but thanks for the suggestion. Maybe later I will submit the intel/2015a toolchain easyconfig.